bin/dredd-hooks-php

```diff
@@ -23,11 +23,48 @@ if (!$loaded) {
     );
 }
 
+// Get options from the command line
+$options = getopt('', [
+    'host:',
+    'port:',
+    'force',
+]);
+
+// Extract parameters from options
+if (array_key_exists('host', $options)) {
+	$host = $options['host'];
+}
+if (array_key_exists('port', $options)) {
+	$port = $options['port'];
+}
+$force = array_key_exists('force', $options);
+
+// Remove parameters from the list of arguments
+foreach($options as $key => $value) {
+	if ($pos = array_search(sprintf('--%s=%s', $key, $value), $argv)) {
+		unset($argv[$pos]);
+		continue;
+	}
+	$posKey = array_search(sprintf('--%s', $key), $argv);
+	$posValue = array_search($value, $argv);
+	unset($argv[$posKey]);
+	if ( 1 == abs($posKey - $posValue)) {
+		unset($argv[$posValue]);
+		continue;
+	}
+}
```

Comment:
@aledeg I am thinking it might be cleaner to parse the hooks like you did for the other cli options / flags and then change `Hooks::loadHooks($hooks)`  rather than passing the entire argv variable in.

What do you think about that?   It was honestly a mistake that I wrote it to accept `$argv` thinking that